src/frontend/components/PreviewTable/index.js

```diff
@@ -90,12 +111,23 @@ const PreviewTable = ({
         <RowComponent
           {...row}
         />
-        <TableCell>
-          <EditButton onClick={() => onEdit(row.id)}>
-            <EditButtonText>Edit</EditButtonText>
-            <EditButtonIcon src={EditIcon} alt="edit" />
-          </EditButton>
-        </TableCell>
+        {onEdit
+         ? <TableCell>
+            <EditButton onClick={() => onEdit(row.id)}>
+              <EditButtonText>Edit</EditButtonText>
+              <EditButtonIcon src={EditIcon} alt="edit" />
+            </EditButton>
+          </TableCell>
+          :<></>
+        }
+        {onSummary
+         ? <TableCell>
+            <SummaryBtn onClick={() => onSummary(row.summary)}>
+              Summary
+            </SummaryBtn>
+          </TableCell>
+          :<></>
```

Comment:
@Samanthatb1 , would it be equivalent to just do `onSummary && ....`. Try it out and tell me how it goes :)

Reply:
Yup its equivalent thanks!